app/services/food_data_service.rb

```diff
@@ -0,0 +1,13 @@
+class FoodDataService
+
+  def get_food_items(url)
+    response = conn.get("foods/search?query=#{params[:q]}")
+    @data = JSON.parse(response.body, symbolize_names: true)
+  end
+
+  def conn
+    connection = Faraday.new(url: "https://api.nal.usda.gov/fdc/v1/") do |f|
+      f.headers["X-API-KEY"] = ENV['food_data_api_key']
+    end
+  end
+end
```

Comment:
I'd say you're on the right track with this refactor. One note about this class as well as `FoodDataFacade`: typically we would expect these to use class methods, so that there isn't a need to instantiate them each time we want to make an API call.